### PR TITLE
update slack webhook to use regex

### PR DESCRIPTION
**Pull request by jinpark**
@hassanabidpk 

uses regex so to get the url
uses the title of the url page

### CODE REVIEW

File: rsvp/migrations/0009_auto_20160421_2121.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rsvp', '0008_auto_20151022_1402'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='reservation',
+            options={'ordering': ['-createdAt']},
```
Comment (jinpark) [inline rsvp/migrations/0009_auto_20160421_2121.py:16]: i havent changed anything but this came up when i created migrations

File: resources/views.py [hunk 2]
```diff
@@ -14,42 +19,38 @@ def index(request):
...
-        except ObjectDoesNotExist:
-            resource = Resource(title=textArray[2],link=link,category=cat)
-            resource.save()
-            return "Resource successfully saved @{}"
-
-    else :
-        return "This resource is not properly formatted @{}"
+def saveResource(user_name, link, title):
+    try:
+        resource = Resource.objects.get(link=link)
```
Comment (jinpark) [inline resources/views.py:51]: this doesnt care about categories

File: resources/views.py [hunk 3]
```diff
@@ -14,42 +19,38 @@ def index(request):
 @csrf_exempt
 def add_resource(request):
     if request.method == 'POST':
-        responseJson = {}
+        response_json = {}
```
Comment (jinpark) [inline resources/views.py:22]: making it more pythonic :D

File: resources/models.py [hunk 4]
```diff
@@ -26,7 +26,7 @@ class Resource(models.Model):
     link = models.URLField()
     description = models.TextField(default='')
     slug = models.SlugField(max_length=100, unique=True)
-    category = models.ForeignKey(Category)
+    category = models.ForeignKey(Category, blank=True, null=True)
```
Comment (jinpark) [inline resources/models.py:29]: making category optional

**Comment by hassanabidpk** [comment]
LGTM - Thanks (I was trying to avoid regex and Beautiful Soap 😄  )

**Pull request merged**